client/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "client",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "apollo-client-preset": "^1.0.8",
```

Comment:
utiliser ca plutot https://www.npmjs.com/package/apollo-boost

Reply:
'apollo-client-preset' removed in favor of 'apollo-boost'